routing/routing.md

```diff
@@ -0,0 +1,55 @@
+# Routing
+
+This document depicts important information about configuring routing for an existing WSO2 product deployment in a
+ container platform.
+ 
+The following will be discussed in detail in the document.
+
+* [Exposing your application to outside](#exposing-your-application-to-outside )
+* [Configuring hostname](#configuring-hostname)
+* [DNS mapping](#dns-mapping)
+* [Configuring SSL](#configuring-ssl)
+* [Tips on Session Affinity](#tips-on-session-affinity)
+
+### Exposing your application to outside
+
+We recommend the use of [official WSO2 product Helm Charts](https://hub.helm.sh/charts/wso2) for production grade WSO2 
+product deployments in Kubernetes environments. [Kubernetes Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+are used to expose WSO2 product deployments in Kubernetes environment. Kubernetes Ingress resources are capable of
```

Comment:
@ThilinaManamgoda let's say `expose WSO2 product services to outside of the Kubernetes environment`. IMO, people may get confused since we use internal Kubernetes Services to expose them to other internal services.

Reply:
Also @ThilinaManamgoda shall we break this paragraph into separate ones. IMO, we could break from the start of the sentence `Kubernetes Ingress resources are capable of...`.

WDYT?

Reply:
+1